admin/js/swonpage-navigation.js

```diff
@@ -236,92 +211,162 @@
         document.body.appendChild(iframe);
     }
 
-    function createItem(item) {
-        if (!sw.inAdminPath() && sw.config.onPage.navigation.ignorePaths.indexOf(item.name) !== -1) {
-            return false;
+    function createElement(page, parentNode) {
```

Comment:
Are you really creating elements here? Or are you creating folder navigation elements?